### DISCUSSION TITLE
CoApp not recognized / not installed?

### DISCUSSION THREAD

**Discussion by Rometach** [aclap-dev/video-downloadhelper#370] [Help and Q&A]
### Your question

It seems to be a common issue but I didn't manage to solve it by myself so here I am. Thank you very much for your help!
DownloadHelper asks me to install the CoApp. 
I installed it with "curl -sSLf https://github.com/aclap-dev/vdhcoapp/releases/latest/download/install.sh | bash" as indicated on the dedicated page, which seemed to work ("CoApp successfuly installed under '~/.local/share/vdhcoapp'."). But the addon keeps telling me I have to install the CoApp (even after I rebooted the browser and the computer).
If I type the command "/opt/vdhcoapp/vdhcoapp --info"
I get "bash: /opt/vdhcoapp/vdhcoapp: Aucun fichier ou dossier de ce nom" (No file or folder with that name)
What do you suggest I try now? :-)

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: fr
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: no.

I'm using DownloadHelper with Firefox under Ubuntu 22.04.4 LTS

**Comment by jcfrog** [comment]
hi,
did you the manual install, downloading the proper package and run ./vdhcoapp install (not as root) after extracting the binaries?

**Comment by Rometach** [comment]
Yes I did! 
"./vdhcoapp install" returns a few lines of installation ending with "VdhCoApp : VdhCoApp is ready to be used", but without changing anything in DH's behaviour or recognition of the CoApp.

**Reply by jcfrog** [reply]
@paulrouget could you have a look?

**Comment by Mattmjs** [comment]
@Rometach , did you ever see a prompt in Firefox when the companion app was installed.

https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#ubuntu-and-flatpak--snap-based-systems

**Reply by Rometach** [reply]
Fixed it, thank you very much. I had jumped to reading the "Linux (all)" part, missing the "Ubuntu (and Flatpak / Snap based systems)" one. Thank you so much for your time, while the answer was right under my nose...
